### ISSUE TITLE
Option to disable ray version check

### ISSUE THREAD

**Issue by danwallach** [modin-project/modin#2104]
**Is your feature request related to a problem? Please describe.**
I need to parse some really big CSV files (100k lines or more), which I'm reading in with Pandas, and I'm processing that data later on with Ray, so Modin seems like a clear win for me. Change one line of code and I get extra speed, right?

However, to use the latest bugfixes from Ray, I need to be running the nightly wheel release. This means that Modin says "nope, wrong version" and I can't use Modin.

Feature request: please have a flag to disable the version pinning. Yes, all bets are off whether this will cause chaos and mayhem, but at least then I've got a shot at getting the speedup from Modin.

**Comment by devin-petersohn** [comment]
Hi @danwallach, we have such a flag, but it may not be clear from the documentation. We are working on the documentation but here is how you enable it.

```python
import os
os.eviron["MODIN_ENGINE"] = "ray"  # do this before importing Modin, can also be set in your environment
import ray
ray.init()  # add params here
import modin.pandas as pd
```

Due to Ray API changes, I have had to edit the source to use nightly wheels. I don't know if it will work, even if you have initialized Ray already due to how we detect if Ray has started. If you're comfortable hacking at source files I can show you where to do that as well.

**Comment by danwallach** [comment]
Great, I'll give that a try. Ideally, I'd like to just pull in Modin as a dependency without having to fork the code.

**Comment by danwallach** [comment]
This turns out to be a bit more complicated. I did as you suggested, but it turns out that Modin insists on calling `ray.init()` itself, even though I've already called it how I want it to be. Also, since Ray 0.8.7, some of the flags have changed (because of course they have), so if I suppress my own call I get this:
```
  File "/Users/dwallach/PycharmProjects/arlo-e2e/.venv/src/modin/modin/pandas/__init__.py", line 113, in _update_engine
    initialize_ray()
  File "/Users/dwallach/PycharmProjects/arlo-e2e/.venv/src/modin/modin/engines/ray/utils.py", line 139, in initialize_ray
    ray.init(
TypeError: init() got an unexpected keyword argument 'plasma_directory'
```

So, feature request: a documented way to say "just use my existing Ray settings".

**Comment by devin-petersohn** [comment]
@danwallach Thanks, that would be a good feature to have. It is difficult to know whether or not Ray has initialized without calling `ray.init` ourselves.

Even in this case, a flag which skips `ray.init` would not work because of changes to the Ray API. #2114 contains the changes necessary to run on Ray nightly if you want to install from there. After Ray 1.0 I think it should be easier to add this flag because the API will be more stable.

**Comment by austinmw** [comment]
@devin-petersohn Hi, I still have the above `_plasma_directory` keyword issue with modin-master. What's the best way to resolve this?

**Comment by devin-petersohn** [comment]
This be resolved as of #2580. There will still be issues if Ray makes breaking changes in their API because we use said API.

Now we don't pin, we only check that the API is compatible. This is the best we can do for this issue because it's a shared API. I don't anticipate any other issues, but please feel free to reopen if this is still an issue!
